src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -1011,6 +1011,7 @@ initialize_query_table()
   emplace_func4_request<query::sdm_sensor_info,                sdm_sensor_info>();
   emplace_sysfs_get<query::hwmon_sdm_serial_num>               ("hwmon_sdm", "serial_num");
   emplace_sysfs_get<query::hwmon_sdm_oem_id>                   ("hwmon_sdm", "oem_id");
+  emplace_sysfs_get<query::hwmon_sdm_board_name>               ("hwmon_sdm", "bd_name");
```

Comment:
I might have missed this earlier, but why do we have sdm specific board info? and why do we have hwmon_sdm directory?

Reply:
@rajkumar-xilinx,
Could you schedule a meeting so that we can go over the use-cases with regards to these changes, especially since they are modifying the various property trees that is being exposed to the user.

Thanks, Stephen